benefits/core/templates/core/includes/sign-out-link.html

```diff
@@ -2,7 +2,7 @@
 {% load i18n %}
 
 {% if authentication %}
-  {% if authentication.required and authentication.logged_in %}
+  {% if authentication.required and authentication.logged_in and authentication.supports_sign_out %}
```

Comment:
```suggestion
  {% if authentication.supports_sign_out and authentication.logged_in %}
```

Reply:
Done in https://github.com/cal-itp/benefits/pull/1462/commits/3fc8ee11708d80beb8e1de32774c4ac07c9ff84b